### PR TITLE
fix(vast): Fix error message for media file validation.

### PR DESCRIPTION
**Pull request by Fan-Xu**
Fix the error messages . for media file validation.

### CODE REVIEW

File: vast/vasttest/vasttest.go [hunk 1]
```diff
@@ -144,3 +152,5 @@ func VerifyVastElementFromFile(t testing.TB, file string, element Validator, exp
 	}
 	VerifyVastElementFromBytes(t, xmlData, element, expectedError)
 }
+
+
```
Comment (xiaopeng0216) [inline vast/vasttest/vasttest.go]: remove extra line

File: vast/vasttest/vasttest.go [hunk 2]
```diff
@@ -114,11 +114,19 @@ func isXmlField(field reflect.StructField) bool {
...
+				if eOk {
+					if ev.Errs[i] == err{
+						return
+					}
+				}else{
+					if expectedError == err {
+						return
+					}
 				}
+
```
Comment (xiaopeng0216) [inline vast/vasttest/vasttest.go]: remove extra line

File: vast/vasttest/vasttest.go [hunk 3]
```diff
@@ -114,11 +114,19 @@ func isXmlField(field reflect.StructField) bool {
 func VerifyVastElementErrorAsExpected(t testing.TB, element Validator, err error, expectedError error) {
 	if err != expectedError {
 		ve, ok := err.(vast.ValidationError)
+		ev,eOk := expectedError.(*vast.ValidationError)
```
Comment (xiaopeng0216) [inline vast/vasttest/vasttest.go]: go lint

Comment (yingchen0706v) [inline vast/vasttest/vasttest.go]: format issue.

File: vast/vasttest/vasttest.go [hunk 4]
```diff
@@ -114,11 +114,19 @@ func isXmlField(field reflect.StructField) bool {
...
 		if ok {
-			for _, err := range ve.Errs {
-				if expectedError == err {
-					return
+			for i, err := range ve.Errs {
+				if eOk {
+					if ev.Errs[i] == err{
+						return
+					}
+				}else{
```
Comment (yingchen0706v) [inline vast/vasttest/vasttest.go]: format issue.

File: vast/vasttest/vasttest.go [hunk 5]
```diff
@@ -114,11 +114,19 @@ func isXmlField(field reflect.StructField) bool {
 func VerifyVastElementErrorAsExpected(t testing.TB, element Validator, err error, expectedError error) {
 	if err != expectedError {
 		ve, ok := err.(vast.ValidationError)
+		ev,eOk := expectedError.(*vast.ValidationError)
 		if ok {
-			for _, err := range ve.Errs {
-				if expectedError == err {
-					return
+			for i, err := range ve.Errs {
+				if eOk {
```
Comment (xiaopeng0216) [inline vast/vasttest/vasttest.go]:
```suggestion
				if eOk && ev.Errs[i] == err {
```

File: vast/vasttest/vasttest.go (hunk 4 above)
Comment (xiaopeng0216) [inline vast/vasttest/vasttest.go]:
```suggestion
				}else if expectedError == err{
```

**Comment by xiaopeng0216** [comment]
LGTM

**Comment by LiuDongxu-Vungle** [comment]
LGTM

**Comment by xiaopeng0216** [comment]
LGTM

**Comment by wenzizone** [comment]
lgtm

**Pull request merged**